src/mono/wasm/build/WasmApp.targets

```diff
@@ -77,12 +80,13 @@
       <MonoAOTCompilerDefaultAotArguments Include="deterministic" />
     </ItemGroup>
     <ItemGroup>
-      <_AotInputAssemblies Include="@(_WasmAssemblies->Distinct())">
+      <_AotInputAssemblies Include="@(_WasmAssembliesInternal->Distinct())" Condition="'%(_WasmAssembliesInternal.InterpOnly)' != 'true'">
```

Comment:
`InterpOnly`  

* Could we use a non-abbreviated name `InterprettedOnly`? Users might need to configure it. 
* Might also want to consider something like `AOTMode=Intepretted`. 

This would look like:

```xml
_AotInputAssemblies 
   Include="@(_WasmAssembliesInternal->Distinct())" 
   Condition="'%(_WasmAssembliesInternal.AOTMode)' != 'Interpretted'">
```

Reply:
> Might also want to consider something like AOTMode=Intepretted.

Does it make sense to have the possible values for `AOTMode` to be as what Android, and iOS have? @lewing @marek-safar @jonathanpeppers

Reply:
What we have now:

https://github.com/xamarin/xamarin-android/blob/accc846e35ccc2b21e2a472d51ab58ed0682643a/src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs#L20-L27

It kind of maps to the native side:

https://github.com/dotnet/runtime/blob/1d9e50cb4735df46d3de0cee5791e97295eaf588/src/mono/mono/mini/jit.h#L46-L72

Reply:
I don't think our integer values matter at build time for what is passed at runtime.

Reply:
And those c# enum names are the strings used with `$(AOTMode)` (or equivalent property name)?

Reply:
Our property is `$(AndroidAotMode)` right now, but we can use a new one in .NET 6 to match other platforms. I'm fine with `$(AOTMode)` if that is what we pick.

Reply:
Sorry, I'm not being clear. What values do you use for `$(AndroidAotMode)`? Are they `$(AndroidAotMode)=normal`, `$(AndroidAotMode)=hybrid` etc, to match the C# enum?

Reply:
It looks like they all map exactly to this C# enum, except for `Intepreter`:

https://github.com/xamarin/xamarin-android/blob/d8df924702a753bbba8d317b3d5e88dde10f7a52/src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs#L96-L122